variable/technology/technologies.yaml

```diff
@@ -1,4 +1,7 @@
 # List of variables related to (energy) technologies
+Active Power Flow|Electricity|Transmission|Line:
```

Comment:
suggest to move this variable to the file `electricity-grid.yaml`

Reply:
I also think so, I'll add a  `power-plant.yaml` file. And,  I was waiting for Sandrine's PR to include this group of variables.